### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by Bob-Murphy**
typos, punctuation and such

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,32 +1,31 @@
...
 
-      The new maps names start with `Antistasi Altis Extender` and the new templates are `CSAT` and `CSAT2`
+      The new maps names start with `Antistasi Altis Extender` and the new templates are `CSAT` and `CSAT2`.
 
-  Assuming everything went well you are now ready to make your own modifications
-  Remember to remove unused content and read thoroughly through the filles while making any edits
-  As a hint all content is added from the config.cpp files located within each of the addons
-  so you can follow that down to the files that govern the different parts like templates and maps etc.
+  Assuming everything went well you are now ready to make your own modifications.
+  Remember to remove unused content and read thoroughly through the filles while making any edits.
```
Comment (jaj22) [inline readme.md]: filles -> files

Comment (Bob-Murphy) [reply readme.md]: done

File: readme.md [hunk 2]
```diff
@@ -51,19 +50,24 @@ Antistasi now supports 3rd party template additions/overwrites.
   Again it's important to read through all the files in the `templates` addon to not miss important steps.
 
 ## Functionality
-- Antistasi now have events that you can listen to and extend existing functionality.
-  A list of all events and their parameters can be found in the in game config under: `A3A >> Events`
-- In addition you can overwrite any of Antistasi functions to add change or remove functionality, this includes full systems (be aware that this is more complex and can break on updates)
-  To do so simply add a function to the `A3A` class of `CfgFunctions.hpp` under the addon functions (you can also replicate the config.cpp to allow this in any other addon)
+- Antistasi now has events that you can listen to and extend existing functionality.
+  A list of all events and their parameters can be found in the in game config under: `A3A >> Events`.
+- In addition you can overwrite any of Antistasis functions to add, change or remove functionality. This includes full systems (be aware that this is more complex and can break on updates).
```
Comment (jaj22) [inline readme.md]: Antistasis -> Antistasi's

Comment (Bob-Murphy) [reply readme.md]: done

File: readme.md [hunk 3]
```diff
@@ -1,32 +1,31 @@
...
+  - Arma 3 Tools  
     
-  Both needs to be run once from steam
+  Both need to be run once from steam.
 
-# How to get started
-- run the `Install.ps1` script and follow the instructions on screen
+# How to get started:
+- Run the `Install.ps1` script and follow the instructions on screen.
 - Run the `Build.ps1` script this will create a folder in the root folder called build and compile your extension into it.
```
Comment (jaj22) [inline readme.md]: Missing a full stop in the middle. Could also make a note that the build script needs to be run again each time you change anything.

File: readme.md [hunk 4]
```diff
@@ -1,32 +1,31 @@
 # Antistasi Extender Example
-This is an example of how you could extend or overwrite antistasi with new maps, templates
+This is an example of how you could extend antistasi with new maps or templates or overwrite existing ones.
```
Comment (jaj22) [inline readme.md]: We're capitalizing Antistasi, right.

File: readme.md (hunk 3 above)
Comment (Bob-Murphy) [reply readme.md]: done

File: readme.md (hunk 4 above)
Comment (Bob-Murphy) [reply readme.md]: done

**Pull request merged**